File: 课程/customCell/customCell/STTg.m
//
//  STTg.m
//  customCell
//
//  Created by kemchenj on 4/28/16.
//  Copyright © 2016 kemchenj. All rights reserved.
//

#import "STTg.h"

@implementation STTg

+(instancetype)tgWithDict:(NSDictionary *)dict
{
    STTg *tg = [[STTg alloc] init];
    [tg setValuesForKeysWithDictionary:dict];
    return tg;
}

@end

File: 课程/test/test/STTg.swift
//
//  File.swift
//  test
//
//  Created by kemchenj on 5/3/16.
//  Copyright © 2016 kemchenj. All rights reserved.
//

import Foundation

class STTg: NSObject {

    var icon: String     = ""
    var title: String    = ""
    var price: String    = ""
    var buyCount: String = ""

    func tgWithDict(dict: NSDictionary) -> STTg {
        let tg = STTg()
        tg.setValuesForKeysWithDictionary(dict as! [String:AnyObject])
        return tg
    }
}